### DISCUSSION TITLE
Организация системы типографики

### DISCUSSION THREAD

**Discussion by monochromer** [doka-guide/platform#281] [Разное]
Как по-вашему должна быть организована типографика на проекте - именование классов, работа с размерами, шрифтами?

**Comment by pepelsbey** [comment]
Я бы пошёл по простому пути:

- Описываем шрифты в отдельных файлах, подключаем WOFF, WOFF2
- Ставим базовый прямой текстовый шрифт на корневой блок, он наследуется вниз
- В местах, где нужно другое начертание или размеры — меняем его
- В местах, где моно-гарнитура — ставим её.

То есть я бы пользовался наследованием и назначением по месту, а не атомарными блоками.

**Reply by pepelsbey** [reply]
Идею можно развить на кастомных свойствах, но это не сильно меняет суть.

**Reply by monochromer** [reply]
Нужно ещё понять, как это будет выглядеть на классах, да ещё и с учтём адаптивности
